### ISSUE TITLE
Not compatible with Foundry 11

### ISSUE THREAD

**Issue by gabrielarantest** [p4535992/foundryvtt-variant-encumbrance-dnd5e#18]
Please kindly make it compatible with Foundry VTT 11.

**Comment by Drakynn** [comment]
Although it was working with FVTT 10, the current error message when you try to install on Ver 11 indicates that you need Ver 9 or older to run the module.

![image](https://github.com/p4535992/foundryvtt-variant-encumbrance-dnd5e/assets/76929992/e7d74f4a-13fe-485b-a63a-f18e7c32d547)

![image](https://github.com/p4535992/foundryvtt-variant-encumbrance-dnd5e/assets/76929992/74735e8a-2a08-4e6e-9e7f-3103bc16b459)

**Comment by discodotgg** [comment]
bump

**Comment by NoriSuii** [comment]
This is holding me from migrating my campaign to v.11, awesome module, please dont die.

**Comment by Nedrapter** [comment]
I'm also in the same position. Is one of the few modules keeping me still on v10.

**Comment by Jefferydo** [comment]
This module makes playing with variant encumbrance feasible, which is magic. It represents the best affordances of VTTs.

**Comment by discodotgg** [comment]
As someone without any developing knowledge, I'll tell you what I did to somewhat get it to work while the definitive fix isn't released. 

In the module.json file from both this and the dependecy modules (library: active effect manager), I changed the lines that regard the version from whatever was there to either 11 or 11.305. Then, I opened Foundry and locked updates for these modules in the Add-on Modules interface. It seems to be working for me thus far.

Regarding the installation, if you just download the module.zip and extract its content into the modules folder, it should also work.
